### ISSUE TITLE
Mimetype: application/x-firmware

### ISSUE THREAD

**Issue by hlechner** [numixproject/numix-icon-theme#566]
Name: --
Mimetype: **application/x-firmware**

Description: **I found it on adwaita theme. I couldn't find a file to check it on nautilus, however it is related to ROM/EEPROM/BIOS update. more details can be check on the following link:** https://mail.gnome.org/archives/commits-list/2015-April/msg00225.html

Icon:
![image](https://cloud.githubusercontent.com/assets/11244067/8095375/0efa9404-0fa4-11e5-8659-8404f357fdbd.png)

**Comment by bil-elmoussaoui** [comment]
@hlechner  I think ```text-x-patch``` would be a good icon for this?

**Comment by hlechner** [comment]
This one I'm not sure, I'm trying to figure out where it is used, if its used as icon inside apps maybe should be good idea keep the IC (integrated circuit) style.

**Comment by hlechner** [comment]
It was implemented on [Gnome 3.18](https://help.gnome.org/misc/release-notes/3.18/):
```
In the past, it has been difficult to update device software (such as your computer’s BIOS) under GNOME. Now this has changed, thanks to a new initiative called the Linux Vendor Firmware Service. This aims to make it possible for hardware manufacturers to distribute firmware updates for GNU/Linux.

The Linux Vendor Firmware Service is fully integrated with GNOME 3.18, and works automatically. If an update becomes available for your device’s hardware, Software will automatically make it available to install the same way as other software updates.

This feature can also be used to update the device software on removable devices, such as monitors, USB hubs and color colorimeters.
```

And the software responsible for, is this one: http://www.fwupd.org/

**Comment by hlechner** [comment]
I think I made a mistake when I created this issue, it's not a mimetype from what i see now.

And also is listed on issue #577 

So I'll close it, but if anyone want to open it again it's ok
